ntbs-service/Pages/Migration/Index.cshtml.cs

```diff
@@ -13,14 +12,19 @@
 using System.ComponentModel.DataAnnotations;
 using System;
 using Hangfire;
+using Microsoft.Extensions.Options;
+using ntbs_service.Properties;
 
 namespace ntbs_service.Pages.Migration
 {
     [Authorize(Policy = "AdminOnly")]
     public class IndexModel : PageModel
     {
-        public IndexModel()
+        private readonly MigrationConfig Config;

```

Comment:
We've been using convention of _config for readonly fields

Reply:
ah good shout. i'll need to find a way for rider to respect that 😂